Countly.js

```diff
@@ -39,22 +41,121 @@ if (Platform.OS.match('android')) {
     Countly.messagingMode.DEVELOPMENT = '2';
 }
 
-// countly initialization
+/**
+ * Initilize Countly
+ *
+ * @deprecated in 20.06.5
+ *
+ * @function Countly.init should be used to initialize countly
+ */
+// countly initialization with config
 Countly.init = async function (serverUrl, appKey, deviceId) {
-    if (deviceId == '') {
-        deviceId = null;
-        Countly.logError('init', "Device Id during init can't be empty string");
+    Countly.logError('init is deprecated, use initWithConfig instead');
+    const countlyConfig = new CountlyConfig(serverUrl, appKey).setDeviceId(deviceId);
+    Countly.initWithConfig(countlyConfig);
+};
+
+/**
+ * Initilize Countly
+ *
```

Comment:
no countlyConfig info

Reply:
I created a new function that has the countlyConfig... line 63